### ISSUE TITLE
Make installation directory (DEST_PATH) available in README

### ISSUE THREAD

**Issue by Dantata** [ihhub/fheroes2#4876]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

I installed fheroes2 on a M1 Mac using homebrew and found that the README.txt file provides the following information:

> As an alternative to the previous step, you can manually copy the subdirectories 'ANIM',
> 'DATA', 'MAPS' and 'MUSIC' (some of them may be missing depending on the version of the
> original game) from the original game directory **to the fheroes2 installation directory**.

I couldn't find details about the installation directory, though. I subsequently found the DEST_PATH variable in the download_demo_version.sh and extract_homm2_resources.sh files:

> if [[ -z "$DEST_PATH" || ! -d "$DEST_PATH" || ! -w "$DEST_PATH" ]]; then
>     if [[ "$(uname 2> /dev/null)" == "Linux" ]]; then
>         DEST_PATH="${XDG_CONFIG_HOME:-$HOME/.local/share}/fheroes2"
>     else
>         **DEST_PATH="$HOME/.fheroes2"**
>     fi
> 

### Describe the possible solution

Adding information about the installation directory (DEST_PATH) in the README file should be more than enough.

**Comment by oleg-derevenetz** [comment]
Hi @Dantata I maintain the fheroes2 package distributed via the MacPorts and handle this by [patching](https://github.com/macports/macports-ports/blob/master/games/fheroes2/files/patch-readme-txt.diff) the README. I believe that the maintainer of the Homebrew package should do the same, because original README is intentionally written in a slightly general way to try to cover more possible platforms and use cases. However, I couldn't figure out who is the maintainer of the corresponding Homebrew [package](https://github.com/Homebrew/homebrew-core/blob/master/Formula/fheroes2.rb) - there is no maintainer information in the package itself and it is updated every time by different people.

**Comment by korikori** [comment]
Hi @oleg-derevenetz, just wanted to chip in that I believe that this problem is not specific to macOS, as I just found out about and installed fheroes2 on Arch Linux earlier today, and did exactly what @Dantata has done -  opened up extract_homm2_resources.sh in order to see the expected installation directory. I feel that the second part of the installation (copying the original data) should be better documented, possibly in its own section in the readme.md or install.md file.

**Comment by oleg-derevenetz** [comment]
@korikori I still believe that it is the responsibility of the maintainer of the relevant package to update the installation instructions in each case. There is no way to fit everything in the generic README file, and currently it contains only instructions for the case when the archive with the game was downloaded from the official website (or from the Releases section here, which is the same). In this case there is an "installation directory" - the directory where this archive was unpacked.

**Comment by Dantata** [comment]
I agree that the README file is supposed to be written in a general way. How about adding a section to the [F.A.Q.](https://github.com/ihhub/fheroes2/wiki/F.A.Q.) then? My point is, it should be easy to find this information, and right now, it isn't. So, people can end up frustrated and unable to install the game.

**Comment by ihhub** [comment]
Moving this topic into the discussion section as no action should be done from the development team.
